CryptoTechReminderSystem.Test/RemindLateDevelopersTests.cs

```diff
@@ -162,5 +119,45 @@ public void CanRemindDevelopersDirectlyWithAMessage()
 
             remindDeveloperSpy.Text.Should().Be("TIMESHEETS ARE GOOD YO!");
         }
+        
+        [Test]
+        public void CanAvoidRemindingLateDevelopersAtTenFifteen()
+        {
+            var remindDeveloperSpy = new RemindDeveloperSpy();
+            var getLateDevelopersStub = new GetLateDevelopersStub();
+            var clock = new ClockStub(
+                new DateTimeOffset(
+                    new DateTime(2019, 03, 01, 10, 15, 0)
+                )
+            );
+            var remindLateDevelopers = new RemindLateDevelopers(getLateDevelopersStub,remindDeveloperSpy, clock);
+            
+            remindLateDevelopers.Execute(new RemindLateDevelopersRequest()
+            {
+                Message = "TIMESHEETS ARE GOOD YO!"
+            });
+
+            remindDeveloperSpy.CountCalled.Should().Be(0);
+        }
+        
+        [Test]
+        public void CanAvoidRemindingLateDevelopersAfterOneThirty()
```

Comment:
Could we use a test-case for this test and the one above?

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/17/commits/03b08f51eff2928f5f0ab3f88905046e24f024a7